src/apps/cacheStatus/options.cpp

```diff
@@ -223,16 +223,17 @@ void COptions::Init(void) {
 
 #ifndef HOST_NAME_MAX
 #define HOST_NAME_MAX 64
```

Comment:
This change is great. Thanks so much for this. Let's correct this mistake as well while we're at it. Host names can be 256 characters long (actually 253, but we can use 256). Can you make that change to the HOST_NAME_MAX variable?